docs/ske/10-integrations/20-port.mdx

```diff
@@ -256,7 +256,13 @@ repository that Port is writing to. An example set up when using FluxCD and the
 from above can be found below.
 
 <details>
-  <summary>Example Flux Resources for Port-Syncer repository</summary>
+  <summary>Example Flux Resources for `port-syncer` repository</summary>
+
+  :::info
+  This example resource depends on a secret called `github-creds` to exist in the
```

Comment:
If we update this secret name to be the same secret name we have in the other guide, it may make it a bit more "portable" (as in, you can follow our configuring flux guide and this will just work).